lib/easypost/services/address.rb

```diff
@@ -17,34 +17,49 @@ def create(params = {})
       wrapped_params[:verify_strict] = params[:verify_strict]
     end
 
-    @client.make_request(:post, 'addresses', MODEL_CLASS, params)
+    response = @client.make_request(:post, 'addresses', params)
+
+    EasyPost::InternalUtilities::Json.convert_json_to_object(response, MODEL_CLASS)
```

Comment:
Question: I am not sure if this way is better since we repeating the same line of code on every function.

Reply:
It's done like this in the Python client. Thought of moving this to the base service class but I don't want us to rely too much on inheritance.

Reply:
We don't have to follow Python lib, we should utilize the features that a language has. I think this change is fine